### DISCUSSION TITLE
Workflow for checking out old branches?

### DISCUSSION THREAD

**Discussion by ozyx** [gitbutlerapp/gitbutler#3761] [Q&A]
Hi there,

I'm wondering if there is a known workflow for checking out an old release branch. For example, in the context of our project, we have some users who are on older versions of the app. But when I apply the old release branch using GitButler, it creates a bunch of merge conflicts for me. The workflow for going "forward" in history is clear to me, but is there a considered workflow for going "backwards", as we need to do this from time to time to create patch releases.

Thanks for your reply and for all the hard work on this awesome tool.

**Comment by Byron** [comment]
Closing as GitButler is a very different app now.
